packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -1020,6 +1020,9 @@ abstract class CustomClipper<T> {
   /// clipped is of the given size.
   T getClip(Size size);
 
+  /// How to clip.
+  ClipOption get clipOption => ClipOption.antiAlias;
```

Comment:
I'm not sure whether I should default to ClipOption.none here. It seems ironical to have no clip as default for a clipper. Currently, only ShapeBorderClipper default to ClipOption.none.

Reply:
yeah the clipping widgets should probably default to `Clip.antiAlias`

Reply:
Ack.